aperturedb/Utils.py

```diff
@@ -617,45 +617,30 @@ def remove_all_objects(self):
 
         cmd = {"constraints": {"_uniqueid": ["!=", "0.0.0"]}}
 
-        queries = [
-            [{"DeleteEntity": cmd}],
-            [{"DeleteImage": cmd}],
-            [{"DeleteVideo": cmd}],
-            [{"DeleteBlob": cmd}],
+        transaction = [
+            {"DeleteImage": cmd},
+            {"DeleteVideo": cmd},
+            {"DeleteBlob": cmd},
             # DeleteDescriptorSet also deletes the descriptors
-            [{"DeleteDescriptorSet": cmd}],
+            {"DeleteDescriptorSet": cmd},
 
             # All the following should be deleted automatically once the
             # related objects are deleted.
             # We keep them here until ApertureDB fully implements this.
-            [{"DeleteBoundingBox": cmd}],
-            [{"DeletePolygon": cmd}],
-            [{"DeleteFrame": cmd}],
+            {"DeleteBoundingBox": cmd},
+            {"DeletePolygon": cmd},
+            {"DeleteFrame": cmd},
+            {"DeleteEntity": cmd},
+            {"GetSchema": {"refresh": True}}
```

Comment:
@venkat-murty assuming all deletes worked, will GetSchema return the new schema after all the deletes?

Reply:
Yes. If we have any delayed deletes (which we want for descriptors), get schema will return that descriptors exist.

Reply:
ok, so we are good for now. Once we have delayed deletes, we revisit the best method for removing all objects.